src/customHooks/useTodoChange.tsx

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable max-len */
+import { useState } from 'react';
+
+export const useTodoChange = (): [number | number[], boolean, ((changingTodosId: number | number[], state: boolean) => void)] => {
+  const [currentTodoId, setCurrentTodoId] = useState<number | number[]>(-1);
+  const [changingState, setChangingState] = useState(false);
+
+  const setTodoChange = (changingTodosId: number | number[], state: boolean) => {
+    setChangingState(state);
+    setCurrentTodoId(changingTodosId);
```

Comment:
and here i wonder if `changingTodosId` is an array,
you do `setCurrentTodoId(changingTodosId);` - so CurrentTodoId becomes an array of ids
what's the intent to have an array for `CurrentTodoId`?
```suggestion
  const setTodoChange = (changingTodosId: number | number[], state: boolean) => {
    setChangingState(state);
    setCurrentTodoId(changingTodosId);
```

Reply:
i mean i get it you want to keep the state of chosen ids may be,
but then just make the naming explicit
prob `selectedTodoIds` instead of `CurrentTodoId`